backend/api/submissions/permissions.py

```diff
@@ -14,6 +15,12 @@ def has_permission(self, source, info):
 
         conference = source.conference
 
+        try:
+            if conference.rankrequest:
+                return True
+        except RankRequest.DoesNotExist:
+            pass
```

Comment:
I wonder if there is a way to make this code look a bit btter

Reply:
don't think so, if the rank request doesn't exist it will throw the exception...